src/v0/destinations/customerio/util.js

```diff
@@ -197,11 +199,15 @@ const groupResponseBuilder = (message) => {
     attributes: payload.attributes || {},
     cio_relationships: [],
   };
-  if (payload.userId) {
-    rawPayload.cio_relationships.push({ identifiers: { id: payload.userId } });
-  } else if (payload.email) {
-    rawPayload.cio_relationships.push({ identifiers: { email: payload.email } });
+  const id = payload?.userId || payload?.email;
+  if (!isDefinedAndNotNull(id)) {
+    throw new InstrumentationError("No Identifier Found. Please provide userId or email.")
```

Comment:
Should we throw the error when there are no identifiers ? I don't see that being done before our changes. Can we confirm if that is necessary ?

Reply:
Removed it to avoid causing any breaking change